cfgrib/__main__.py

```diff
@@ -40,29 +40,62 @@ def selfcheck() -> None:
 
 @cfgrib_cli.command("to_netcdf")
 @click.argument("inpaths", nargs=-1)
-@click.option("--outpath", "-o", default=None)
-@click.option("--cdm", "-c", default=None)
-@click.option("--engine", "-e", default="cfgrib")
-def to_netcdf(inpaths, outpath, cdm, engine):
-    # type: (T.List[str], str, str, str) -> None
+@click.option(
+    "--outpath", "-o", default=None,
+    help="Filename of the output netcdf file."
+)
+@click.option(
+    "--cdm", "-c", default=None,
+    help='Coordinate model to translate the grib coordinates to.'
+)
+@click.option(
+    "--engine", "-e", default="cfgrib",
+    help='xarray engine to use in xarray.open_dataset.'
+)
+@click.option(
+    "--backend-kwargs-json", "-b", default=None,
+    help=(
+        'Backend kwargs used in xarray.open_dataset.'
+        'Can either be a JSON format string or '
+        'the path to JSON file'
+    )
+)
+def to_netcdf(inpaths, outpath, cdm, engine, backend_kwargs_json):
+    # type: (T.List[str], str, str, str, str) -> None
     import xarray as xr
-
-    import cf2cdm
-
     # NOTE: noop if no input argument
     if len(inpaths) == 0:
         return
 
     if not outpath:
         outpath = os.path.splitext(inpaths[0])[0] + ".nc"
 
+    if backend_kwargs_json is not None:
+        import json   # only import if used
+        try:
+            # Assume a json format string
+            backend_kwargs = json.loads(backend_kwargs_json)
+        except json.JSONDecodeError:
+            # Then a json file
+            with open(backend_kwargs_json, 'r') as f:
+                backend_kwargs_json = json.load(f)
```

Comment:
Hi @EddyCMWF, shouldn't this be **backend_kwargs**= json.load(f) here?

Reply:
Thanks, was a typo from my refactoring